eZ/Publish/Core/Repository/Values/MultiLanguageValueTrait.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/**
+ * File containing the eZ\Publish\API\Repository\Values\MultiLanguageValueTrait trait.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\Repository\Values;
+
+/**
+ * @internal Meant for internal use by Repository, type hint against API object instead.
+ */
+trait MultiLanguageValueTrait
```

Comment:
Even if internal, wouldn't it be better to split it to two traits? (`MultiLanguageValueNameTrait`, `MultiLanguageValueDescriptionTrait`). Right now it seems to be rather more like a abstract base class than single responsibility trait.